demo/test.c

```diff
@@ -2214,6 +2214,70 @@ static int test_s_mp_toom_sqr(void)
    return EXIT_FAILURE;
 }
 
+static int test_mp_read_write_binaries(void)
```

Comment:
separate tests test_mp_read_write_ubin, test_mp_read_write_sbin

Reply:
> minad added this to the v1.2.0 milestone 26 minutes ago

Always that pressure! ;-)